pkg/quota/api/validation/validation_test.go

```diff
@@ -0,0 +1,9 @@
+package validation
+
+import (
+	"testing"
+)
+
+func TestClusterQuota(t *testing.T) {
+
```

Comment:
Intention was good, execution a bit worse ;) @deads2k mind updating in a followup?